src/config/entities/configuration.ts

```diff
@@ -53,4 +53,7 @@ export default () => ({
   safeTransaction: {
     useVpcUrl: process.env.USE_TX_SERVICE_VPC_URL?.toLowerCase() === 'true',
   },
+  features: {
+    humanDescription: process.env.FF_HUMAN_DESCRIPTION === 'true',
```

Comment:
To keep it consistent, can we use `?.toLowerCase()` here also?

Reply:
done